core/src/main/codegen/templates/Parser.jj

```diff
@@ -1822,23 +1822,47 @@ List<SqlNode> SelectList() :
 SqlNode SelectItem() :
 {
     SqlNode e;
+    SqlIdentifier columnAlias;
+    final List<SqlNode> columnAliases = new ArrayList();
+    final Span s = span();
     final SqlIdentifier id;
```

Comment:
Can we merge the parse of `AS` token, and the `span()` position is not correct.

Reply:
@danny0405 I was always confusion on how to correct use `span()`. Is there some documentation you know that explains how `span()` (and position) work?

Reply:
The implementation code for `span()` is `Span.of(getPos())`, which contains the position of the last visited token. For example, if i have a parse block:
```java
<AS> ... span()
```
The span would contains the position of the `<AS>` token.

Reply:
Done.